### PR TITLE
Fix jabba.rb formula

### PR DESCRIPTION
**Pull request by ghost**
- [ X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ X ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ X ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jabba.rb [hunk 1]
```diff
@@ -29,13 +30,15 @@ def install
       system "go", "build", "-ldflags", ldflags, "-o", bin/"jabba"
       prefix.install_metafiles
     end
+    ENV["JABBA_HOME"] = "$HOME/.jabba"
```
Comment (SMillerDev) [inline Formula/jabba.rb:33]: This won't do anything

File: Formula/jabba.rb [hunk 2]
```diff
@@ -29,13 +30,15 @@ def install
...
   end
 
   test do
     ENV["JABBA_HOME"] = testpath/"jabba_home"
     system bin/"jabba", "install", "openjdk@1.14.0"
     jdk_path = shell_output("#{bin}/jabba which openjdk@1.14.0").strip
     assert_match 'openjdk version "14',
-                 shell_output("#{jdk_path}/Contents/Home/bin/java -version 2>&1")
+    shell_output("#{jdk_path}/Contents/Home/bin/java -version 2>&1")
+    assert_match "#{HOMEBREW_PREFIX}/bin/jabba", shell_output("which jabba")
```
Comment (SMillerDev) [inline Formula/jabba.rb:42]: There is no need to test the homebrew setup

**Comment by gromgit** [comment]
@michaelcusick, what exactly are you fixing in this formula? (That would've been nice to know from the start.)

**Comment by michaelcusick** [comment]
There are issues with jabba working at all. See https://github.com/shyiko/jabba/issues/138.

**Pull request closed without merging**